src/pages/other-services/resources-and-toolkits/covid-19-kit-for-parents.tsx

```diff
@@ -0,0 +1,103 @@
+import Link from "next/link";
+
+import { List } from "@/components/Services/list";
+
+export default function ResourcesAndToolkits() {
+  const covidBookletList = (
+    <List
+      items={[
+        "families and organisations with practical tips, activities and links to resources that are focused on women and children and young people’s health and wellbeing.",
+        "offers guidance, support and information for women whose families are in isolation."
+      ]}
+    />
+  );
+
+  const body = (
+    <>
+      <div className="flex justify-center bg-[--starick-white] px-6 py-2 md:px-32">
+        <div className="w-fit">
+          <br></br>
+          <p>
+            Starick’s COVID-19 resource booklets were developed in consultation
+            with children, young people and families living in refuges or
+            receiving support from FDV services.{" "}
+          </p>
+          <br></br>
+          <p>The booklet provides</p>
+          {covidBookletList}
+          <br></br>
+          <p>
+            As we move into living with COVID-19, this booklet can be used more
+            broadly to support families and children living in refuges and
+            outreach services
+          </p>
+          <br></br>
+          <p>
+            The booklets provide direct links to online resources. Please make
+            sure that supervision of children and young people online is age
+            appropriate.
+          </p>
+          <br></br>
+          <br></br>
+        </div>
+      </div>
+    </>
+  );
+
+  return (
+    <>
+      <div>
+        <img
```

Comment:
It's better to use [`next/image`](https://nextjs.org/docs/pages/api-reference/components/image) instead as this is a strictly better version which supports optimisation, etc.